Nové zprávy pro příchod a odchod členů

janchaloupka opened the pull request:
- Bot už nebude odesílat DMs novým uživatelům, protože Discord má integrovaný (a mnohem lepší) onboarding
 - Upozornění když někdo odejde
 - Rozdělení upozornení o příchodu na nového uživatele a na vrácení bývalého člena
 - Upozornění mají víc tvarů, takže je to víc fun a random

janchaloupka committed:
Zákázání DMs pro nové uživatele

Díscord už má lepší, interaktivní onboarding, takže je to zbytečný.

janchaloupka committed:
Přepracovány notifikace příchozích a odchozích lidí

Misha12 reviewed (approved):
Je to OK. Ještě by mi přišlo nice to have přidat zprávu o tom, že uživatel byl kicknut/zabanován, ale nutně nemusí být

janchaloupka committed:
Odstranění příkazů na svátky

Nemyslím si že to někdy někdo použil

janchaloupka committed:
Odstranění synchronizace bodů z Kachny

Myslím že je to zbytečný. Bobánkov nemá s Kahcnou nic moc společnýho a taky se to dlouho nepoužívalo.

janchaloupka committed:
Odtranění přidávání rolí přes reakce

Discord to umí sám

Khubajsn reviewed (approved):
Bylo by hezký ty leave notifikace vyřešit tak, aby fungovaly i pro jiný způsob odejití (smazání účtu) nebo když zrovna bot neposlouchá na eventy. Ale beru, že to tak není vyřešený ani pro uvítací notifikace..
